src/lib/dialog/dialog-directives.ts

```diff
@@ -0,0 +1,49 @@
+import {Directive} from '@angular/core';
+import {MdDialog} from './dialog';
+
+
+/**
+ * Button that will close the current dialog.
+ */
+@Directive({
+  selector: 'button[md-dialog-close]',
```

Comment:
Also add `mat-` prefixed selectors (will be used for material1 compatibility mode).